com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.PostProcess.cs

```diff
@@ -3019,6 +3019,9 @@ TextureHandle PaniniProjectionPass(RenderGraph renderGraph, HDCamera hdCamera, T
                     else
                         passData.paniniProjectionCS.EnableKeyword("UNITDISTANCE");
 
+                    if (m_EnableAlpha)
```

Comment:
You may want to add the else statement to disable the keyword. 
It will avoid issues when doing two cameras back to back with one using alpha and the other not. The second camera would then not use the proper keyword.
(It's not currently possible to have different color buffer types per camera but with render graph it should be soon a possibility so might as well do it right now)

Reply:
Will do - also there are a few other instances in the post stack that only enable that keyword, like Stop NaNs pass: 
https://github.com/Unity-Technologies/Graphics/blob/a85221a204200b45175d26a4ea95195b5ed2e51f/com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.PostProcess.cs#L546-L547

I can also fix that up for the other instances of it in this PR.